Gemfile

```diff
@@ -12,12 +12,8 @@ gem "rack-canonical-host"
 gem "rails", "~> 6.0.0"
 
 # Front-endy
-gem "autoprefixer-rails"
```

Comment:
I spot a postcss config using [postcss-preset-env](https://github.com/csstools/postcss-preset-env) that handles autoprefixing.